core/src/main/java/jenkins/util/SystemProperties.java

```diff
@@ -308,6 +311,49 @@ public static Boolean optBoolean(String name) {
         return v == null ? null : Boolean.parseBoolean(v);
     }
 
+    /**
+     * Determines the duration value of the system property with the specified name.
+     * @param name property name.
+     * @return the property value as a duration.
+     * @since TODO
+     */
+    @CheckForNull
+    public static Duration getDuration(@NonNull String name) {
+        return getDuration(name, null);
+    }
+
+    /**
+     * Determines the duration value of the system property with the specified name, or a default value.
+     * @param name property name.
+     * @param defaultValue a default value
+     * @return the property value as a duration.
+     * @since TODO
+     */
+    @Nullable
```

Comment:
Last I checked, you cannot use `@Nullable` any more: https://github.com/jenkinsci/plugin-pom/issues/1063

Reply:
Ok, will use `@CheckForNull` instead.

Reply:
Actually it seems to be ok?

Reply:
:shrug: maybe there was a regression that was later fixed.